code/edgeconsent/build.gradle

```diff
@@ -243,31 +210,35 @@ task platformFunctionalTestJacocoReport(type: JacocoReport, dependsOn: "createPh
     ])
 
     reports {
-        xml.enabled true
-        csv.enabled false
-        html.enabled true
+        xml.required = true
+        csv.required = false
+        html.required = true
     }
 }
 
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+}
 
 dependencies {
     // Adobe Mobile SDK Core
     implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
 
     testImplementation "androidx.test.ext:junit:${rootProject.ext.junitVersion}"
     testImplementation "org.mockito:mockito-core:${rootProject.ext.mockitoCoreVersion}"
-    testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5'
+    testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.12.7'
     testImplementation 'org.powermock:powermock-api-mockito2:2.0.0'
     testImplementation 'org.powermock:powermock-module-junit4:2.0.0'
     testImplementation 'org.json:json:20180813'
 
     androidTestImplementation "androidx.test.ext:junit:${rootProject.ext.junitVersion}"
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     // Using older version of PowerMock as it supports minimum Android API < 26
     // the root issue appears to be a dependency with Objenesis in Mockito-Core
     // where Objenesis 2 requires minimum Android API 26
     androidTestImplementation 'org.powermock:powermock-module-junit4:1+'
-    androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5'
+    androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.12.7'
+    androidTestImplementation project(path: ':edgeconsent')
```

Comment:
It shouldn't be required to explicitly include the project module for the instrumented tests. Can you remove this line?